exercises/poker/canonical-data.json

```diff
@@ -0,0 +1,317 @@
+{
+  "exercise": "poker",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "it should be able to pick the winning poker hand",
+      "cases": [
+        {
+          "description": "single hand always wins",
+          "property": "best_hand",
```

Comment:
`_` isn't an allowed character in https://github.com/exercism/x-common/blob/master/canonical-schema.json#L117, so I would suggest `bestHand`. Maybe even `bestHands` since there can be more than one.